### DISCUSSION TITLE
Caching the previous page

### DISCUSSION THREAD

**Discussion by webdeb** [vercel/next.js#17597] [Ideas]
Just had that idea in mind, that when you navigate from a list like view, eg. an infinite scroll to some page/x. And go back, the user looses the last state. Would be nice, if the user could see the previous page in the same state as before.

However its just an idea so I thought this might be the right place to ask, what you think about it and what are the technical implications.

**Comment by IRediTOTO** [comment]
did you try swr? :)

**Reply by IRediTOTO** [reply]
https://swr.now.sh/
